prac_05/state_names.py

```diff
@@ -0,0 +1,49 @@
+"""
+CP1404/CP5632 Practical
+State names in a dictionary
+File needs reformatting
+"""
+
+CODE_TO_NAME = {"QLD": "Queensland",
+                "NSW": "New South Wales",
+                "NT": "Northern Territory",
+                "WA": "Western Australia",
+                "ACT": "Australian Capital Territory",
+                "VIC": "Victoria",
+                "TAS": "Tasmania"}
+
+MAX_KEY_LEN = len(max(CODE_TO_NAME, key=len))
+MIN_KEY_LEN = len(min(CODE_TO_NAME, key=len))
+MAX_VALUE_LEN = len(CODE_TO_NAME[max(CODE_TO_NAME, key=len)])
+MIN_VALUE_LEN = len(CODE_TO_NAME[min(CODE_TO_NAME, key=len)])
+
+
+
+def main():
+    """Turn short state codes to full names, then prints the entire key"""
+    decode_short_state()
+    print_state(CODE_TO_NAME)
+
+
+def decode_short_state():
```

Comment:
Could the function name be better? Try saying out loud "This function will decode short state", does it flow well?

Reply:
Yeah I do need to practice better naming appreciate it.